src/GraphQL/FromServicesAttribute.cs

```diff
@@ -0,0 +1,20 @@
+using GraphQL.Types;
+using GraphQL.Utilities;
+
+namespace GraphQL
+{
+    /// <summary>
+    /// Specifies that the method argument should be pulled from <see cref="IResolveFieldContext.RequestServices"/>.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Parameter)]
```

Comment:
AllowMultiple = false

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.attributeusageattribute.allowmultiple?view=net-6.0

> true if more than one instance is allowed to be specified; otherwise, false. The default is false.